scripts/insert_aggkey_sc.py

```diff
@@ -0,0 +1,45 @@
+import sys
+import os
+
+sys.path.append(os.path.abspath("tests"))
+from shared_tests import *
+from utils import prompt_user_continue_or_break
+from consts import *
+from brownie import accounts, KeyManager, network
+
+AUTONOMY_SEED = os.environ["SEED"]
+cf_accs = accounts.from_mnemonic(AUTONOMY_SEED, count=10)
+DEPLOYER_ACCOUNT_INDEX = int(os.environ.get("DEPLOYER_ACCOUNT_INDEX") or 0)
+
+DEPLOYER = cf_accs[DEPLOYER_ACCOUNT_INDEX]
+print(f"DEPLOYER = {DEPLOYER}")
+
+
+def main():
+    print()
```

Comment:
what's this?

Reply:
I went too fast copy pasting :sweat_smile: